src/underactuated/pyplot_visualizer.py

```diff
@@ -31,11 +26,12 @@ class PyPlotVisualizer(LeafSystem):
         state.
     '''
     
-    def __init__(self, draw_rate=0.0333, facecolor=[1, 1, 1]):
+    def __init__(self, draw_rate=60, facecolor=[1, 1, 1]):
```

Comment:
Good to fix this misnomer! This was mentioned in another comment, but maybe passing timestep rather than rate would be more natural. Users would be more comfortable using the same timestep for the logger sample timestep and the visualizer timestep (and they wouldn't worry about picking a timestep that was exactly 1/rate without numerical error). Most of the uses of rate in this class take 1/rate anyway (for periodic publish timestep, resampling step size, and animation interval).

Reply:
switched to timestep.